### PR TITLE
Note on communication module for SuperTeams

### PR DESCRIPTION
**Pull request by mrshu**
### Please describe your change in one or two sentences

* Add a note on communication module provided by the OC to the teams as
      part of the SuperTeam challenge.

### Please explain why do you think this change should be in the rules

* To make the SuperTeam games more manageable (especially from the organizational/referee perspective) and at the same time to foster inter-team collaboration and communication.

### CODE REVIEW

File: rules.adoc [hunk 1]
```diff
@@ -1045,21 +1045,27 @@ used.
...
 During the competition days of the International Competition (as well as before
 the event) the team members are responsible for checking all relevant
-information published by the Soccer Organizational Committee, General Chairs,
+information published by the Soccer Organization Committee, General Chairs,
 or any other RoboCup official.
 
 There will also be a SuperTeam competition, in which various people from around
 the world share their robots in one "SuperTeam" and play against other
-SuperTeams on a so called "Big Field". The full rules of this challenge can be found at
+SuperTeams on a so called "Big Field".  The full rules of this challenge can be found at
```
Comment (dbscoach) [inline rules.adoc:1058]: This space might go if you're touching the file again.

File: rules.adoc [hunk 2]
```diff
@@ -1045,21 +1045,27 @@ used.
...
 https://robocupjuniortc.github.io/soccer-rules/master/superteam_rules.html
 
+NOTE: {++To make the communication between multiple robots in a single
+"SuperTeam" easier and the games more manageable, the Organization Committee
+will provide each team a communication module. Each team will be expected to
+interface with this module using a single 2.54mm GPIO pin. More details will be
+provided by the Organization Committee shortly before the competition.++}
+
```
Comment (dbscoach) [inline rules.adoc:1066]: This is only a "will" for the international level and much more of a "may" for local/regional/superregional levels.

Comment (mrshu) [reply rules.adoc:1066]: Agreed, but this is in the "international" section after all :)

File: rules.adoc (hunk 1 above)
Comment (mrshu) [reply rules.adoc:1058]: Thanks -- will try to fix it

File: rules.adoc (hunk 2 above)
Comment (dbscoach) [reply rules.adoc:1066]: As it is it's not really making communication easier - that I think starts at UART or something like that. At the moment it only makes start/stop control easier.

Comment (dbscoach) [reply rules.adoc:1066]: Agreed on intl. section, did not spot that in the diff...

File: rules.adoc [hunk 3]
```diff
@@ -29,7 +29,7 @@ by the RoboCupJunior Soccer Committee. The English version of these
 rules has priority over any translations.
 
 Teams are advised to check the RoboCupJunior Soccer site
-https://junior.forum.robocup.org/ for OC (Organizational Committee) procedures
+https://junior.forum.robocup.org/ for OC (Organization Committee) procedures
```
Comment (dbscoach) [inline rules.adoc:32]: Do we want to rename what I put as soccer league committee 2022 at the top to just Organization Committee as well for consistency?

File: rules.adoc (hunk 2 above)
Comment (mrshu) [reply rules.adoc:1066]: That's fair -- I'd be happy to take any suggestions you might have here. I was taking a bit longer view here which might not be there yet :slightly_smiling_face:

**Review by dbscoach — APPROVED** [review]
I think this has no show-stoppers and is sufficiently "wait for details" to offer all necessary flexibility.

File: rules.adoc (hunk 3 above)
Comment (mrshu) [reply rules.adoc:32]: It's a good question but I think we can do that once the concept of a Committee is a bit more clear (i.e. not now :slightly_smiling_face: )

File: rules.adoc (hunk 2 above)
Comment (dbscoach) [reply rules.adoc:1066]: {++To make SuperTeam games more manageable now and make communication between multiple robots in a SuperTeam easier in the future the Organization Committee will provide each team with a communication module. Each team will be expected to interface with this module using a single 2.54mm GPIO pin now and the Organiyation committee plans on extending this to using UART for more complex applications in future years. More details will be provided by the Organization Committee shortly before the competition.++}

Comment (dbscoach) [reply rules.adoc:1066]: I don't know if that makes the the sentence overly complicated but I'd like to provide as much "look forward" as to increase what ideas teams can generate and contribute in the future. Feel free to simplify, I'm not very good at changing existing phrasings.

Comment (mrshu) [reply rules.adoc:1068]: Thanks @dbscoach!

Would you mind taking another look?

**Review by dbscoach — APPROVED** [review]

**Pull request merged**